Transfer/Sources/ScheduleTransferSectionData.swift

```diff
@@ -0,0 +1,129 @@
+
+#if !COCOAPODS
+import Common
+#endif
+
+import HyperwalletSDK
+
+enum ScheduleTransferSectionHeader: String {
+    case button, destination, foreignExchange, notes, summary
+}
+
+protocol ScheduleTransferSectionData {
+    var scheduleTransferSectionHeader: ScheduleTransferSectionHeader { get }
+    var rowCount: Int { get }
+    var title: String? { get }
+    var cellIdentifier: String { get }
+}
+
+extension ScheduleTransferSectionData {
+    var rowCount: Int { return 1 }
+    var title: String? { return scheduleTransferSectionHeader
+        != ScheduleTransferSectionHeader.button ?
+            "schedule_transfer_section_header_\(scheduleTransferSectionHeader.rawValue)".localized() : nil }
+}
+
+struct ScheduleTransferDestinationData: ScheduleTransferSectionData {
+    var scheduleTransferSectionHeader: ScheduleTransferSectionHeader { return .destination }
+    var cellIdentifier: String { return TransferDestinationCell.reuseIdentifier }
+    var transferMethod: HyperwalletTransferMethod
+
+    init(transferMethod: HyperwalletTransferMethod) {
+        self.transferMethod = transferMethod
+    }
+}
+
+struct ScheduleTransferForeignExchangeData: ScheduleTransferSectionData {
+    var rows = [(title: String, value: String)]()
+    var scheduleTransferSectionHeader: ScheduleTransferSectionHeader { return .foreignExchange }
+    var rowCount: Int { return rows.count }
+    var cellIdentifier: String { return ScheduleTransferForeignExchangeCell.reuseIdentifier }
+    var foreignExchanges: [HyperwalletForeignExchange]
+
+    init(foreignExchanges: [HyperwalletForeignExchange]) {
+        self.foreignExchanges = foreignExchanges
+
+        for (index, foreignExchange) in foreignExchanges.enumerated() {
+            if let sourceAmount = foreignExchange.sourceAmount,
+                let sourceCurrency = foreignExchange.sourceCurrency,
+                let destinationAmount = foreignExchange.destinationAmount,
+                let destinationCurrency = foreignExchange.destinationCurrency,
+                let  rate = foreignExchange.rate {
+                let sourceAmountFormatted = sourceAmount.format(with: sourceCurrency)
+                let destinationAmountFormatted = destinationAmount.format(with: destinationCurrency)
+
+                let rateFormatted = String(format: "%@ = %@",
+                                           "1".format(with: sourceCurrency),
+                                           rate.format(with: destinationCurrency))
+
+                rows.append((title: "transfer_fx_sell_confirmation".localized(), value:sourceAmountFormatted))
+                rows.append((title: "transfer_fx_buy_confirmation".localized(), value: destinationAmountFormatted))
+                rows.append((title: "transfer_fx_rate_confirmation".localized(), value: rateFormatted))
+                if !foreignExchanges.isLast(index: index) {
+                    rows.append((title: "", value: ""))
```

Comment:
use `title: nil, value: nil ` instead of `title: "", value: ""` by making title and value as optional

Reply:
Sure